app/js/components/tabs/tabcomponents/patientComponent.jsx

```diff
@@ -71,24 +72,24 @@ class PatientComponent extends Component {
       }
     }
     // add appropriate birthdate constraints to the search parameters
-    if (startDate && endDate) {
+    if (startDate && endDate) {  
+      // coerce the values of start date and end date to numbers for comparison    
+      let startYear = Number(startDate.split('-')[0]);
```

Comment:
I thought this part was already merged in this pull request https://github.com/openmrs/openmrs-owa-cohortbuilder/pull/93